gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -162,14 +162,17 @@ class AkkaGrpcPlugin implements Plugin<Project> {
 
         project.task("printProtocLogs") {
             doLast {
-                Files.lines(logFile).forEach { line ->
-                    if (line.startsWith("[info]")) logger.info(line.substring(7))
-                    else if (line.startsWith("[debug]")) logger.debug(line.substring(7))
-                    else if (line.startsWith("[warn]")) logger.warn(line.substring(6))
-                    else if (line.startsWith("[error]")) logger.error(line.substring(7))
+                if (Files.exists(logFile)) {
+                    Files.lines(logFile).forEach { line ->
+                        if (line.startsWith("[info]")) logger.info(line.substring(7))
+                        else if (line.startsWith("[debug]")) logger.debug(line.substring(7))
+                        else if (line.startsWith("[warn]")) logger.warn(line.substring(6))
+                        else if (line.startsWith("[error]")) logger.error(line.substring(7))
+                    }
                 }
             }
         }
+        project.getTasks().getByName("printProtocLogs").dependsOn("generateProto")
```

Comment:
Besides adding this task dependency, it's also a good idea to add `logFile` to the task outputs of `generateProto`, so that gradle can properly retrigger the task if the file is missing or stale. Be careful that `generateProto` was added as part of `project.afterEvaluate`, so you'll have to do the same here.

Taking a huge step back, since `generateProto` by default outputs everything under `build/generated/source/proto/<sourceSet>`, it might be a better idea to actually move the `logFile` to be over there, so we don't have to mess with its outputs and we also avoid overwrites from multiple source sets (which may be another bug right now?)

Reply:
That sounds like a sensible improvement, but I think I'd like to keep things less ambitious in this PR for now :)